airflow/dags/operations/docs_operations.py

```diff
@@ -0,0 +1,98 @@
+import os
+import logging
+import shutil
+from pathlib import Path
+from zipfile import ZipFile
+
+import requests
+from lxml import etree
+
+import common.hooks as hooks
+
+Logger = logging.getLogger(__name__)
+
+
+def list_documents(sps_package):
+    """
+    Lista todos os XMLs dos SPS Packages da lista obtida do diretório do XC.
+
+    list sps_packages: lista com os paths dos pacotes SPS no diretório de processamento
+    dict sps_packages_xmls: dict com os paths dos pacotes SPS e os respectivos nomes dos
+        arquivos XML.
+    """
+    Logger.debug("list_documents IN")
```

Comment:
Uma dúvida: estas mensagens "list_documents IN" são de fato úteis?

Reply:
estamos utilizando agora, so a carater de Reestrutura e "placeholder" da função ainda, em futuros Pr esse log pode ser removido